tests/misc/iterable_sections/sections-ram.ld

```diff
@@ -1,2 +1,3 @@
 ITERABLE_SECTION_RAM(test_ram, 4)
 ITERABLE_SECTION_RAM(test_ram2, 4)
+ITERABLE_SECTION_ROM(test_ram_named, 4)
```

Comment:
Should probalby be `ITERABLE_SECTION_RAM`

Reply:
fixed